gprofiler/perf.py

```diff
@@ -18,21 +20,35 @@
 logger = get_logger_adapter(__name__)
 
 
+@functools.lru_cache(maxsize=1)
+def perf_path() -> str:
+    return resource_path("perf")
+
+
 class PerfProcess:
     _dump_timeout_s = 5
     _poll_timeout_s = 5
 
-    def __init__(self, frequency: int, stop_event: Event, output_path: str, is_dwarf: bool, extra_args: List[str]):
+    def __init__(
+        self,
+        frequency: int,
+        stop_event: Event,
+        output_path: str,
+        is_dwarf: bool,
+        inject_jit: bool,
+        extra_args: List[str],
+    ):
         self._frequency = frequency
         self._stop_event = stop_event
         self._output_path = output_path
-        self._extra_args = extra_args
         self._type = "dwarf" if is_dwarf else "fp"
+        self._inject_jit = inject_jit
+        self._extra_args = extra_args + (["-k", "1"] if self._inject_jit else [])
```

Comment:
why is it needed for inject-jit ?

Reply:
It needs to correlate timestamps in the `jitdump` file with those from `perf.data`. So both need to use the same clock